packages/admin/src/table/filterbar/FilterBarActiveFilterBadge.tsx

```diff
@@ -0,0 +1,72 @@
+import { createStyles, Theme, Typography, WithStyles, withStyles } from "@material-ui/core";
+import * as React from "react";
+import { useForm } from "react-final-form";
+
+export type CometAdminFilterBarActiveFilterBadgeClassKeys = "hasValueCount";
+
+export const dirtyFieldsCount = (values: Record<string, any>, registeredFields: string[] = []) => {
+    let count = 0;
+    Object.entries(values).forEach(([fieldName, fieldValue]) => {
+        if (!fieldValue) {
+            return;
+        }
+
+        const isRegisteredField = registeredFields.includes(fieldName);
+        if (Array.isArray(fieldValue) && isRegisteredField) {
+            count += fieldValue.length;
+        } else if (typeof fieldValue === "object") {
+            if (Object.keys(fieldValue).length === 2 && "min" in fieldValue && "max" in fieldValue && isRegisteredField) {
+                count++;
+            } else {
+                count += dirtyFieldsCount(fieldValue, registeredFields);
+            }
+        } else if (isRegisteredField) {
+            count++;
+        }
+    });
+
+    return count;
+};
+
+const styles = (theme: Theme) =>
+    createStyles<CometAdminFilterBarActiveFilterBadgeClassKeys, FilterBarActiveFilterBadgeProps>({
+        hasValueCount: {
+            backgroundColor: `${theme.palette.grey[100]}`,
+            textAlign: "center",
+            borderRadius: "4px",
+            padding: "0 5px",
+            height: "20px",
+        },
+    });
+
+export interface FilterBarActiveFilterBadgeProps {
+    values: Record<string, any>;
+    calcNumberDirtyFields?: (values: Record<string, any>, registeredFields: string[]) => number;
```

Comment:
the new argument needs to be added for all calcNumberDirtyFields definitions (FilterBarPopoverFilterProps)

Reply:
thats true :) will add it :)